### DISCUSSION TITLE
I want to set login on my TERMUX app

### DISCUSSION THREAD

**Discussion by Nueldes** [termux/termux-app#3419] [General]
How do I fix this

apt update && apt install git -y && git clone https://github.com/htr-tech/termux-login.git && cd termux-login && chmod +x * && sh install.sh
Ign:1 https://mirrors.sahilister.in/termux/termux-main stable InRelease
Ign:1 https://mirrors.sahilister.in/termux/termux-main stable InRelease
Ign:1 https://mirrors.sahilister.in/termux/termux-main stable InRelease
Err:1 https://mirrors.sahilister.in/termux/termux-main stable InRelease
  Something wicked happened resolving 'mirrors.sahilister.in:https' (7 - No address associated with hostname)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: Failed to fetch https://mirrors.sahilister.in/termux/termux-main/dists/stable/InRelease  Something wicked happened resolving 'mirrors.sahilister.in:https' (7 - No address associated with hostname)
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
git is already the newest version (2.40.1-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Cloning into 'termux-login'...                                  fatal: unable to access 'https://github.com/htr-tech/termux-login.git/': Could not resolve host: github.com

**Comment by sylirre** [comment]
Fix your internet connection
